### PR TITLE
String example extensions

### PR DESCRIPTION
**Pull request by rogermc2**
Example using GL.Text
Common made into library.

### CODE REVIEW

File: examples/common/src/main.adb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+procedure Main is
```
Comment (flyx) [inline examples/common/src/main.adb]: What is this file for? You should declare `common` as `library project` instead so that it does not need a `main`.

Comment (rogermc2) [reply examples/common/src/main.adb]: It shouldn't be there. I'm removing it.
It must have got automatically generated by GPS.
Still, I should have noticed it.

Comment (rogermc2) [reply examples/common/src/main.adb]: `main` deleted.
Bad styles corrected.

**Comment by rogermc2** [comment]
Thanks Flyx.
This marks a milestone in my development.

**Comment by rogermc2** [comment]
Thanks Flyx.
This marks a milestone for me on my OpenGL experience.
However, I'm wondering about the use of this develop branch.
At first I didn't relalize and merged upstream/master which of course caused problems.
When I merged upstream/master things seem to have recovered.

**Comment by rogermc2** [comment]
There were problems due to removal of opengl shared.
I have submitted a separate PR to fix those.

**Comment by rogermc2** [comment]
You did mention that you intended to use a  `develop branch`.
Does this mean that I should create new branches using
`git checkout -b <new branch> upstream develop`?

**Comment by flyx** [comment]
I believe the correct syntax is `git checkout -b <new branch> upstream/develop` but yes.

**Comment by rogermc2** [comment]
That's what I meant. Typo. Sorry and thanks.

**Pull request merged**